### PR TITLE
Fixes issue #15131. Guns will now fire point blank correctly.

### LINKED ISSUE #15131: Point blank fire is bugged
If you are one tile away, doesn't matter where you aim, it will always trigger the execution, BUT, it will only work if you are aiming for mouth. If you aim for anything else, it will trigger it, but even after the whole bar is completed it will cancel as if you walked away

### PR DESCRIPTION
**Pull request by KazeEspada**
Fixes #15131

### CODE REVIEW

File: code/modules/projectiles/gun.dm [hunk 1]
```diff
@@ -147,8 +147,9 @@
 		return
 
 	if(flag)
-		handle_suicide(user, target, params) //
-		return
+		if(user.zone_sel.selecting == "mouth")
```
Comment (tkdrg) [inline code/modules/projectiles/gun.dm]: will conflict with #15177, not sure if that is an issue though

Comment (KazeEspada) [inline code/modules/projectiles/gun.dm]: Merge which ever first.

Comment (tkdrg) [inline code/modules/projectiles/gun.dm]: that one is merged, so pleasedchange this line

**Comment by phil235** [comment]
Need to change "user.zone_sel.selecting" to "user.zone_selected"

**Pull request merged**